packages/x-charts/src/LineChart/seriesConfig/tooltipPosition.ts

```diff
@@ -0,0 +1,32 @@
+import type { TooltipItemPositionGetter } from '../../internals/plugins/models/seriesConfig/tooltipItemPositionGetter.types';
+
+const tooltipItemPositionGetter: TooltipItemPositionGetter<'line'> = (params) => {
+  const { series, identifier, axesConfig } = params;
+
+  if (!identifier || identifier.dataIndex === undefined) {
+    return null;
+  }
+  const itemSeries = series.line?.series[identifier.seriesId];
+
+  if (series.line == null || itemSeries == null) {
```

Comment:
Same as for the bar x other charts. I wonder if it makes sense to abstract the common logic from these methods across different chart. I am asking because the logic is very similar, so if something needs to be fixed in one chart, would be great to have the logic for the other types of charts in the same place. Could be overabstraction tough, so I will leave it to you.

Reply:
I first considered getting directly the `itemSeries` but bar chart requires extra information (the staking groups)

If we move this logic lot of stuff will be impacted :)